src/main/java/org/mitre/hapifhir/MedMorphToCIBMTR.java

```diff
@@ -1,10 +1,200 @@
+      httpPost.setHeader("Content-Type", "application/fhir+json");
+      httpPost.setHeader("Authorization", "Bearer " + authToken);
+
+      JSONObject bundleRequestBody = new JSONObject();
+      bundleRequestBody.put("resourceType", "Bundle");
+      bundleRequestBody.put("type", "transaction");
+      bundleRequestBody.put("entry", getObservationEntries(observationEntries, resourceId));
+
+      StringEntity stringEntity = new StringEntity(bundleRequestBody.toString());
+      httpPost.setEntity(stringEntity);
+      httpClient.execute(httpPost);
+    } catch (Exception e) {
+      return;
+    }
+  }
+
+  private JSONArray getObservationEntries(List<BundleEntryComponent> observationEntries, String resourceId) {
+    JSONArray entryArray = new JSONArray();
+
+    for (BundleEntryComponent entry : observationEntries) {
+      JSONObject observationObject = new JSONObject();
+      JSONObject requestObject = new JSONObject();
+      requestObject.put("method", "POST");
+      requestObject.put("url", "Observation");
+      observationObject.put("request", requestObject);
+
+      Observation observation = (Observation)entry.getResource();
+      JSONObject observationResourceObject = new JSONObject();
+      observationResourceObject.put("resourceType", "Observation");
+      observationResourceObject.put("meta", getMeta());
+      observationResourceObject.put("subject", (new JSONObject()).put("reference", "Patient/" + resourceId));
+      observationResourceObject.put("effectiveDateTime", observation.getEffectiveDateTimeType().dateTimeValue().getValue());
+      CodeableConcept code = observation.getCode();
+      Coding coding = code.getCoding().get(0);
+      JSONObject codingObject = new JSONObject();
+      codingObject.put("system", coding.getSystem());
+      codingObject.put("code", coding.getCode());
+      codingObject.put("display", coding.getDisplay());
+      JSONObject codeObject = new JSONObject();
+      codeObject.put("coding", (new JSONArray()).put(codingObject));
+      observationResourceObject.put("code", codeObject);
+      Quantity quantity = observation.getValueQuantity();
+      JSONObject quantityObject = new JSONObject();
+      quantityObject.put("value", quantity.getValue());
+      quantityObject.put("unit", quantity.getUnit());
+      quantityObject.put("system", quantity.getSystem());
+      quantityObject.put("code", quantity.getCode());
+      observationResourceObject.put("valueQuantity", quantityObject);
+      observationObject.put("resource", observationResourceObject);
+      entryArray.put(observationObject);
+    }
+
+    return entryArray;
+  }
+
+  private JSONObject getMeta() {
+    JSONObject metaObject = new JSONObject();
+    JSONArray securityArray = new JSONArray();
+    JSONObject securityObject = new JSONObject();
+    securityObject.put("system", "http://cibmtr.org/codesystem/transplant-center");
+    securityObject.put("code", "rc_12001");
```

Comment:
I think the 12001 here is the sample ccn? So this should be using the `ccn` field